documentation/identifiers.md

```diff
@@ -78,65 +88,94 @@ There are many different kinds of organisations that give, receive or benefit fr
-    * 'GB-NIC-' for a [charity registered in Northern Ireland](http://www.charitycommissionni.org.uk/) 
-    
-    along with the registered number.
+[org-id.guide](http://org-id.guide) ranks identifier lists by relevance and quality to help you pick the best identifier. 
 
-3.  **Educational establishment (UK)**
+```eval_rst
 
-    If you have an educational establishment for the organisation, use the Registration Agency Code:
-      * 'GB-EDU-' for a school, university or other educational establishment in England and Wales along with the [EduBase](http://www.education.gov.uk/edubase/home.xhtml) URN identifier. 
-      * 'GB-UKPRN-' for a school, university or other educational establishment in Scotland or Northern Ireland along with the [UK Register of Learning](https://www.ukrlp.co.uk/) UKPRN number.
+.. hint:: Relevance and quality defined: 
 
-4. **Other registered number**
+  * Relevance: are you likely to find the organisation you are looking for in this list?
+  * Quality: are the identifiers in this list stable and linked to open, accessible contextual data, and can they be easily mapped to other identifiers.
+```
 
-    If you have a registered number from some other scheme, including overseas registrars, check the [IATI Organisation Registration Agency Codelist](http://iatistandard.org/codelists/OrganisationRegistrationAgency/) for a Registration Agency Code to use. If the Registration Agency Code you need is not listed, [contact the support team](http://www.threesixtygiving.org/contact/).
+Search on [org-id.guide](http://org-id.guide) for identifier sources for [UK organisations](http://org-id.guide/?structure=&coverage=GB&subnational=&sector=), [UK charities](http://org-id.guide/?structure=charity&coverage=GB&sector=), or [any other organisation type](http://org-id.guide/).
 
-5. **No registered number**
+### Commonly used identifier lists
 
-    If you do not have any external registration numbers for the organisation, use your 360Giving prefix and any internal identifier* you have for this organisation.
-    
-    *If you use a database that records details of organisations in a separate lookup table, this may provide an identifier you can use. 
+The following identifier lists are often used in 360Giving publication. They are listed here in rough order of priority (e.g. if you already know the company number, use this in preference to the charity number).
+
+* [GB-COH](http://org-id.guide/list/GB-COH) - UK Company Number
+* [GB-CHC](http://org-id.guide/list/GB-CHC), [GB-SC](http://org-id.guide/list/GB-SC), [GB-NIC](http://org-id.guide/list/GB-NIC) - Charity Numbers
+* [GB-EDU](http://org-id.guide/list/GB-EDU) and [GB-UKPRN](http://org-id.guide/list/GB-UKPRN) - Education establishments
+* [GB-LAE](http://org-id.guide/list/GB-LAE) (England), [GB-LAS](http://org-id.guide/list/GB-LAS) (Scotland), [GB-PLA](http://org-id.guide/list/GB-PLA) (Wales)  - Local authorities
+
```

Comment:
We should add GB-REV here and explain why we recommend not to use it

Reply:
Any others we don't want to encourage use of - wasn't there an example in Northern Ireland?

Reply:
I think what you mean is that there's currently an issue with the Northern Ireland Charity Register as charities are still being registered, so many will not have an GB-NIC id, only an HMRC number which is what all were using previously.

Reply:
Thanks Bob! This is the example we are referring to, but we also saw England and Wales charities using a REV number as their charity identifier, and we don't want them to do this (look at the Dulverton trust for examples). Also, some users were not aware of REV numbers as all so we need to add it to the documentation please (see #211)